index.html

```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+		   <!-- Purva Gharat card START -->
```

Comment:
Just fix this minor indentation then you should be good to go!

Reply:
Indentation removed ! Thank You.

Reply:
Thank you, merged your pr!